tests/Mutator/Number/DecrementIntegerTest.php

```diff
@@ -150,6 +190,24 @@ public function provideMutationCases(): array
 if (count($a) < -1) {
     echo 'bar';
 }
+PHP
+                ,
+            ],
+            'It does decrement when compared against a variable function' => [
```

Comment:
could you please restore test removed in the previous commit and make sure that it does **not** decrement `0` to `-1` in this case:

```php
if (0 > count($a)) {	
    echo 'bar';	
}
```

Reply:
In that case the following should also not be allowed:

```diff
<?php

- if (count($a) < 0) {
+ if (count($a) < -1) {
    echo 'bar';
}
```

I'm thinking we can probably simplify this whole thing and just say: 'if count is compared to 0 in any way, don't mutate it.'

Reply:
> if count is compared to 0 in any way, don't mutate it

yes, since result of `count()` is strictly not negative